public/level/tiled-level/test-bumpy.json

```diff
@@ -0,0 +1,327 @@
+                }],
+         "opacity":1,
+         "type":"objectgroup",
+         "visible":true,
+         "x":0,
+         "y":0
+        }, 
+        {
+         "draworder":"topdown",
+         "id":2,
+         "name":"spawner",
+         "objects":[
+                {
+                 "height":0,
+                 "id":1,
+                 "name":"player",
+                 "point":true,
+                 "rotation":0,
+                 "type":"",
+                 "visible":true,
+                 "width":0,
+                 "x":320,
+                 "y":318
+                }, 
+                {
+                 "height":0,
+                 "id":3,
+                 "name":"goal",
+                 "point":true,
+                 "rotation":0,
+                 "type":"",
+                 "visible":true,
+                 "width":0,
+                 "x":3965,
+                 "y":-384
+                }, 
+                {
+                 "height":0,
+                 "id":27,
+                 "name":"coin",
+                 "point":true,
+                 "rotation":0,
+                 "type":"",
+                 "visible":true,
+                 "width":0,
+                 "x":936,
+                 "y":280
+                }, 
+                {
+                 "height":0,
+                 "id":28,
+                 "name":"coin",
+                 "point":true,
+                 "rotation":0,
+                 "type":"",
+                 "visible":true,
+                 "width":0,
+                 "x":680,
+                 "y":236
+                }, 
+                {
+                 "height":0,
+                 "id":29,
+                 "name":"coin",
+                 "point":true,
+                 "rotation":0,
+                 "type":"",
+                 "visible":true,
+                 "width":0,
+                 "x":1480,
+                 "y":156
+                }, 
+                {
+                 "height":0,
+                 "id":30,
+                 "name":"coin",
+                 "point":true,
+                 "rotation":0,
+                 "type":"",
+                 "visible":true,
+                 "width":0,
+                 "x":1208,
+                 "y":180
+                }],
+         "opacity":1,
+         "type":"objectgroup",
+         "visible":true,
+         "x":0,
+         "y":0
+        }],
+ "nextlayerid":6,
+ "nextobjectid":31,
+ "orientation":"orthogonal",
+ "renderorder":"right-down",
+ "tiledversion":"1.10.2",
+ "tileheight":32,
+ "tilesets":[
+        {
+         "columns":15,
+         "firstgid":1,
+         "image":"..\/tileset\/demo-tileset.png",
+         "imageheight":96,
+         "imagewidth":480,
+         "margin":0,
+         "name":"tiles",
+         "spacing":0,
+         "tilecount":45,
+         "tileheight":32,
+         "tilewidth":32
+        }],
+ "tilewidth":32,
+ "type":"map",
+ "version":"1.10",
+ "width":30
+}
```

Comment:
i guess we should delete this file?

Reply:
hmmm thought that would have been removed when I merged main 🤔 it's gone now!

Reply:
its still here :thinking:

Reply:
ok should be gone now, I didn't commit the deleted file!